posts/2020/02/my-unexpected-dive-into-open-source-python.md

```diff
@@ -0,0 +1,61 @@
+<!--
+.. title: My Unexpected Dive into Open-Source Python
+.. slug: my-unexpected-dive-into-open-source-python
+.. date: 2020-02-17 16:38:07 UTC-06:00
+.. tags: Labs, PyData
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+![Header Image](/images/codesprite-comics/My-Unexpected-Dive-Into-Open-Source-Python-Illustration.png)
+*Header illustration by author, Mars Lee*
```

Comment:
This isn't showing up. Maybe just need a blank line below `-->` (just guessing)?

Reply:
Added the blank space now